### PR TITLE
[717938] New Blueliv ThreatCompass integration

### PR DESCRIPTION
**Pull request by oylosoar**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added a new integration for Blueliv ThreatCompass

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [x] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No (new integration)

## Must have
- [x] Tests
- [x] Documentation 

## Demisto Partner?
- [x] The title must be in the following format: **[717938] New Blueliv ThreatCompass integration**

### CODE REVIEW

**Comment by liorblob** [comment]
Hi @oylosoar , I've left some comments. Can we schedule a short demo to see the test playbook run etc.?
Thanks.

**Comment by oylosoar** [comment]
> Hi @oylosoar , I've left some comments. Can we schedule a short demo to see the test playbook run etc.?
> Thanks.

Hi @liorblob , I can not see any comments. Today I'm able to have a call from 8 to 9 CEST or 16:30 to 17:30 is this ok for you?
Thank you

**Comment by liorblob** [comment]
@oylosoar sorry, submitted it now. I'm available at 16:30. To whom should I send the zoom invite? I can do so via email or slack(rather not expose email/zoom links here).

**Comment by oylosoar** [comment]
@liorblob perfect, I'm looking at the comments. I have just written you a Slack message. 
Thank you!

File: Packs/BluelivThreatCompass/Integrations/BluelivThreatCompass/BluelivThreatCompass.py [hunk 1]
```diff
@@ -6,10 +6,7 @@
 import urllib3
 from datetime import datetime
 
-urllib3.disable_warnings()
```
Comment (liorblob) [inline Packs/BluelivThreatCompass/Integrations/BluelivThreatCompass/BluelivThreatCompass.py]: why delete this?

Comment (oylosoar) [reply Packs/BluelivThreatCompass/Integrations/BluelivThreatCompass/BluelivThreatCompass.py]: I added it inside main function

**Pull request merged**